gitbootcamp.rst

```diff
@@ -18,7 +18,7 @@ Git Bootcamp and Cheat Sheet
    desire a different approach.
 
 
-In this section, we'll go over some commonly used Git commands that are
+In this section, we will go over some commonly used Git commands that are
```

Comment:
Personally, I don't think there's an issue with using contractions, only that within a given article, the form chosen for a term should be consistent. Since ``gitbootcamp`` uses ``we'll``, it should not use ``we will``. However, either option is okay.

Reply:
Actually gitbootcamp.rst was using both.

Reply:
@maggyero Oh, interesting. In that case, we should probably use whichever form was encountered first and stick with it.